Libraries/Network/__tests__/XMLHttpRequestBase-test.js

```diff
@@ -54,6 +54,25 @@ describe('XMLHttpRequestBase', function(){
 		expect(xhr.readyState).toBe(xhr.OPENED);
     });
 
+    it('should expose responseText correctly', function() {
+		expect(xhr.responseType).toBe('');
```

Comment:
Please don’t test for this in a test for `responseType`. Either set it, or trust the default (which should be tested in its own test block).